.buildscript/settings.xml

```diff
@@ -1,14 +1,9 @@
 <settings>
   <servers>
-    <server>
-      <id>ossrh-snapshot</id>
-      <username>${env.CI_DEPLOY_USERNAME}</username>
-      <password>${env.CI_DEPLOY_PASSWORD}</password>
-    </server>
-    <server>
-        <id>ossrh</id>
-        <username>${env.CI_DEPLOY_USERNAME}</username>
-        <password>${env.CI_DEPLOY_PASSWORD}</password>
-    </server>
-</servers>
+      <server>
+          <id>ossrh</id>
+          <username>${env.CI_DEPLOY_USERNAME}</username>
+          <password>${env.CI_DEPLOY_PASSWORD}</password>
+      </server>
+  </servers>
```

Comment:
You sent me credentials for `ossrh` only and not `ossrh-snapshot` so I deleted the XML for the latter.

Reply:
the identifier is arbitrary, it is only a relation between the pom.xml and settings.xml.

what matters is to have a proper Sonatype token

Reply:
OK. If it's arbitrary, I will revert this change.